### PR TITLE
AGR-2708 - DRAFT, do not merge

### PR DESCRIPTION
**Pull request by lpalbou**
I spin up a test site: http://35.153.103.5:2992 but this site will go down in the next few days.

I only activated reactome pathways & reactions, not GO-CAMs, and this will only work on human genes at the moment. I still have to tweak a little bit for non human genes through orthologs.

### CODE REVIEW

**Comment by sibyl229** [comment]
Hi @lpalbou thanks a lot for creating a PR and deploying a testing environment! 

I tried to checkout the code locally. But I encountered an error with `npm install`. I will attach the error I get below. What version of Node are you using? 

It seems that CI test also failed. Could you look into that please?

```
(base) ML4876-SGAO:agr_ui sgao$ nvm use
Found '/Users/sgao/study/agr/agr_ui/.nvmrc' with version <lts/erbium>
Now using node v12.21.0 (npm v6.14.11)
(base) ML4876-SGAO:agr_ui sgao$ npm install

> sleep@3.0.0 install /Users/sgao/study/agr/agr_ui/node_modules/sleep
> node-gyp rebuild

  CXX(target) Release/obj.target/node_sleep/sleep.o
../sleep.cc:6:11: error: no member named 'Handle' in namespace 'v8'
using v8::Handle;
      ~~~~^
../sleep.cc:42:30: error: too few arguments to function call, single argument 'context' was not specified
  sleep(info[0]->Uint32Value());
        ~~~~~~~~~~~~~~~~~~~~ ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8.h:2707:3: note: 'Uint32Value' declared here
  V8_WARN_UNUSED_RESULT Maybe<uint32_t> Uint32Value(
  ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8config.h:368:31: note: expanded from macro 'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
../sleep.cc:54:31: error: too few arguments to function call, single argument 'context' was not specified
  usleep(info[0]->Uint32Value());
         ~~~~~~~~~~~~~~~~~~~~ ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8.h:2707:3: note: 'Uint32Value' declared here
  V8_WARN_UNUSED_RESULT Maybe<uint32_t> Uint32Value(
  ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8config.h:368:31: note: expanded from macro 'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
../sleep.cc:62:52: error: too few arguments to function call, single argument 'context' was not specified
    Nan::New<FunctionTemplate>(Sleep)->GetFunction());
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8.h:6126:3: note: 'GetFunction' declared here
  V8_WARN_UNUSED_RESULT MaybeLocal<Function> GetFunction(
  ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8config.h:368:31: note: expanded from macro 'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
../sleep.cc:64:53: error: too few arguments to function call, single argument 'context' was not specified
    Nan::New<FunctionTemplate>(USleep)->GetFunction());
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8.h:6126:3: note: 'GetFunction' declared here
  V8_WARN_UNUSED_RESULT MaybeLocal<Function> GetFunction(
  ^
/Users/sgao/Library/Caches/node-gyp/12.21.0/include/node/v8config.h:368:31: note: expanded from macro 'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
5 errors generated.
make: *** [Release/obj.target/node_sleep/sleep.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/sgao/.nvm/versions/node/v12.21.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/Users/sgao/.nvm/versions/node/v12.21.0/bin/node" "/Users/sgao/.nvm/versions/node/v12.21.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/sgao/study/agr/agr_ui/node_modules/sleep
gyp ERR! node -v v12.21.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sleep@3.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sleep@3.0.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sgao/.npm/_logs/2021-04-07T20_58_07_830Z-debug.log

```

File: src/containers/genePage/index.js [hunk 1]
```diff
@@ -207,12 +231,6 @@ const GenePage = ({geneId}) => {
             focusGeneId={data.id}
           />
         </Subsection>
-        <Subsection title={GENETIC_INTERACTIONS}>
```
Comment (sibyl229) [inline src/containers/genePage/index.js:210]: This would delete the genetic interactions table.

File: src/containers/genePage/index.js [hunk 2]
```diff
@@ -102,7 +112,7 @@ const GenePage = ({geneId}) => {
           icon={<SpeciesIcon inNav scale={0.5} species={data.species.name} />}
           truncateName
         >
-          <SpeciesName>{data.species.name}</SpeciesName>
+          <i>{data.species.name}</i>
```
Comment (sibyl229) [inline src/containers/genePage/index.js:115]: Is this intentional?

File: src/containers/genePage/index.js [hunk 3]
```diff
@@ -76,7 +71,22 @@ const GenePage = ({geneId}) => {
     return null;
   }
 
-  const genomeLocation = getSingleGenomeLocation(data.genomeLocations);
```
Comment (sibyl229) [inline src/containers/genePage/index.js:79]: I don't believe this change is intentional. It seems to have reverted the code to an earlier state.

**Review by sibyl229 — CHANGES_REQUESTED** [review]
@lpalbou could you also take a look at the `src/containers/genePage/index.js`? I think many of the changes here aren't intentional (ie removing the Genetic interactions table). Maybe it was accidentally saved with an earlier version of the file?

**Comment by lpalbou** [comment]
The rebase wasn't successful and is starting to be too messy. Just creating another branch/PR from the current master branch.

**Pull request closed without merging**